scripts/word_freq_table.js

```diff
@@ -0,0 +1,54 @@
+window.addEventListener("DOMContentLoaded", function () {
+    function checkAllVisible() {
+        const table = document.getElementById(`table_${this.dataset.instance}`);
+        const tableBodies = table.getElementsByClassName("table_body");
```

Comment:
FYI the class `table_body` seems pretty synthetic at first glace. I think this is needed because we don't set a `<thead>` for the header row here:

https://github.com/DistributedProofreaders/dproofreaders/blob/0abe4844c429ae04cb8eb27184e56dae6d7b5e0d/tools/project_manager/word_freq_table.inc#L183

Per the [html 5 spec](https://html.spec.whatwg.org/multipage/parsing.html#parsing-main-intable), this means that the browser inserts a `<tbody>` for it to keep the table structure.

Not sure if that's really worth it, but _I think_ we could probably remove the `table_body` class and use `table.getElementsByTagName("tbody")` if we fixed this but curious about your thoughts.

Reply:
I had wondered if getElementsByTagName("tbody") would be better but was a bit wary of it. The way the table is constructed seems a bit obscure to me so I didn't want to change it. I'll do what you suggest.